tests/api/test_dynamo.py

```diff
@@ -360,3 +362,42 @@ def test_query_jobs_sort_order(tables):
 
     response, _ = dynamo.query_jobs('user1')
     assert response == table_items
+
+
+def test_decimal_conversion(tables):
+    table_items = [
+            {
+                'job_id': 'job1',
+                'job_type': 'RTC_GAMMA',
+                'name': 'name1',
+                'user_id': 'user1',
+                'status_code': 'status1',
+                'request_time': '2000-01-03T00:00:00+00:00',
+                'float_value': 30.04,
+            },
+            {
+                'job_id': 'job2',
+                'job_type': 'RTC_GAMMA',
+                'name': 'name1',
+                'user_id': 'user1',
+                'status_code': 'status1',
+                'request_time': '2000-01-02T00:00:00+00:00',
+                'float_value': 0.0,
+            },
+            {
+                'job_id': 'job3',
+                'job_type': 'INSAR_GAMMA',
+                'name': 'name2',
+                'user_id': 'user1',
+                'status_code': 'status1',
+                'request_time': '2000-01-01T00:00:00+00:00',
+                'float_value': 0.1,
+            },
+        ]
+    for item in table_items:
+        tables['jobs_table'].put_item(Item=util.convert_floats_to_decimals(item))
+
+    response, _ = dynamo.query_jobs('user1')
+    assert response[0]['float_value'] == Decimal('30.04')
+    assert response[1]['float_value'] == Decimal('0.0')
+    assert response[2]['float_value'] == Decimal('0.1')
```

Comment:
This final section could be a separate `test_convert_floats_to_decimals` test in test_util.py ? Then in test_dynamo.py it's sufficient to check that `put_item` doesn't raise an exception?